src/app/blog/index.html

```diff
@@ -4,6 +4,21 @@
 	windowTitle="SKY UX Blog"
 	showTableOfContents="true">
 
+	<article class="blog-post">
+		<header>
+			<stache-page-anchor>
+				Design update — June 2021
+			</stache-page-anchor>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+			<h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-lookup/docs/lookup">Lookup enhancements</a></h4>
+				<p>We added functionality to the lookup component that allows users to add new items on the fly and browse the search matches. With these enhancements we will be deprecating <a href="https://developer.blackbaud.com/skyux-lookup/docs/autocomplete">autocomplete</a> and <a href="https://developer.blackbaud.com/skyux-select-field/docs/select-field">select field</a>; any use of those should be replaced with lookup.</p>
+			<h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-text-editor/docs/text-editor">Text editor</a></h4>
+				<p>We added a text editor that allows users to enter formatted text.</p>
```

Comment:
Should we mention that this is a beta release? Or just rely on the note on the component docs?

Reply:
Ran this question by @Blackbaud-KerryCampbell, and we're going to rely on the note in the component docs.